service/test/BatchServerTest.cpp

```diff
@@ -47,6 +48,11 @@
 #include <sys/mman.h>
 #include <unistd.h>
 
+#include <cstring>    // for strlen(), strncmp()
+#include <string>     // for std::string
+#include <sstream>    // for std::stringstream
+#include <iostream>   // for std::streambuf, std::cerr, std::flush
```

Comment:
```suggestion
#include <cstring>
#include <string>
#include <sstream>
#include <iostream>
```

These comments are unnecessary.  Please remove them.